### ISSUE TITLE
Allow users to input fractional feerate

### ISSUE THREAD

**Issue by alecalve** [wizardsardine/liana#1480]
I can input 6 sats or 7 sats but not 6.5 (the replaced tx fee rate is much lower than 6 sats/vB).

It’s especially useful for low feerate txs, if a tx is at 1 sats/vB, currently there’s no choice but to double the fee paid.

![image](https://github.com/user-attachments/assets/80ae0fab-52f0-4a88-af8d-f4c441a4709e)

**Comment by jp1ac4** [comment]
Thanks for reporting this issue.

Allowing fractional feerates (for both new spends and RBFs) is certainly feasible and we will consider this for a future release.

**Comment by MisterCyp** [comment]
Hello Team,

Thank you for your great work. I’m using Liana as my main wallet, and I’m very happy with it.

One feature that would be extremely useful would be the ability to use fractional values for the fee rate.

**Comment by nondiremanuel** [comment]
We could try include feerate < 0 together with the fractional ones.

**Comment by pythcoiner** [comment]
> We could try include feerate < 0 together with the fractional ones.

I dont think so, iirc from a code pov it's different scopes, but anyway fractional feerate must be done first

One think that should be defined is what unit we store in our data model (we DO NOT want to deal with float), iirc I had looked at this last year and 1/100 sat/vb (stored in `u32`) was convenient for conversion, meaning our minimal fee increment would be 0.01 sat/vb
